src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -63,8 +85,7 @@ function startApplication() {
     render(<Application router={router} />, applicationElement);
 }
 
-const translationPromise = Requester.get('/admin/v2/translations?locale=en')
-    .then((response) => setTranslations(response));
+const translationPromise = Requester.get('/admin/v2/translations?locale=en');
 
 const configPromise = Requester.get('/admin/v2/config').then((response) => {
```

Comment:
To avoid more strange behavior as before I would say we do everything in this callback in the one for `Promise.all`.